### PR TITLE
update IAM tutorial to use channel configuration

### PR DESCRIPTION
**Pull request by spoorthipujariadobe**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/tutorials/ajo/create-channel-configuration.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+4. Click the **Create channel configuration** button in the top right.
+
+| ![Create new channel configuration](assets/channel-new.png?raw=true) |
+| :---: |
+| **Create new channel configuration** |
+
+5. Give the channel configuration an identifying name and description (**1**). 
+6. From the **Select channel** drop down, select **In-app messaging** (**2**).
+7. Under the In-app messaging settings section, select the checkbox for **iOS** (**3**) and enter the tutorial app's bundle identifier in the **App id** field under **iOS** (**4**).
+8. Under the In-app messaging settings section, select the checkbox for **Android** (**5**) and enter the tutorial app's bundle identifier in the **App id** field under **Android** (**6**).
```
Comment (sbenedicadb) [inline docs/tutorials/ajo/create-channel-configuration.md:28]: should we clarify that these two steps are only if they are targeting those platforms?

File: docs/tutorials/dc/create-tag-property.md [hunk 2]
```diff
@@ -123,4 +123,4 @@ In this section we will create a mobile property which contains the configuratio
 | :---: |
 | **View install instructions** |
 
-For the next part of the tutorial, we will show how to author an in-app message in **Adobe Journey Optimizer**.
\ No newline at end of file
+For the next part of the tutorial, we will show how to create the channel configuration in **Adobe Journey Optimizer**.
```
Comment (sbenedicadb) [inline docs/tutorials/dc/create-tag-property.md:126]: do we also need to update this part in the datastream doc, to point to creating a tag property next (which is a new order for this update)?

**Review by sbenedicadb — APPROVED** [review]
a couple small updates but looks good

File: docs/tutorials/dc/create-tag-property.md (hunk 2 above)
Comment (spoorthipujariadobe) [reply docs/tutorials/dc/create-tag-property.md:126]: The datastream page didn't have a pointer to the surface page. I added one to point to tag property step

File: docs/tutorials/ajo/create-channel-configuration.md [hunk 3]
```diff
@@ -24,9 +24,8 @@ In order for you to properly select the destination app of an in-app message in
 
 5. Give the channel configuration an identifying name and description (**1**). 
 6. From the **Select channel** drop down, select **In-app messaging** (**2**).
-7. Under the In-app messaging settings section, select the checkbox for **iOS** (**3**) and enter the tutorial app's bundle identifier in the **App id** field under **iOS** (**4**).
-8. Under the In-app messaging settings section, select the checkbox for **Android** (**5**) and enter the tutorial app's bundle identifier in the **App id** field under **Android** (**6**).
-9. Hit the **Submit** (**7**) button.
+7. Under the In-app messaging settings section, select the platform you're targetting. For iOS, check **iOS** (**3**) and enter the tutorial app's bundle identifier in the **App id** field under **iOS** (**4**). For Android, check **Android** (**5**) and enter the tutorial app's app id in the **App id** field under **Android** (**6**).
```
Comment (sbenedicadb) [inline docs/tutorials/ajo/create-channel-configuration.md:27]: nit: typo - `targeting`

**Review by sbenedicadb — APPROVED** [review]
one tiny typo to fix then it's good to merge 👍

**Pull request merged**